CMakeLists.txt

```diff
@@ -111,10 +112,9 @@ endif ()
 
 
 # Set the default namespace
-if (OSL_NAMESPACE)
-    add_definitions ("-DOSL_NAMESPACE=${OSL_NAMESPACE}")
-endif ()
-message(STATUS "Setting Namespace to: ${OSL_NAMESPACE}")
+set (PROJ_NAMESPACE "${${PROJ_NAME}_NAMESPACE}")
```

Comment:
I'm less worried about breaking things with the namespace change (though maybe I should be), but rather that there was a pretty significant refactoring of the cmake scripts in master after it diverged from 1.8, so there is no easy clean merge to backport just the namespace change in isolation. I was really hoping not to need to mess with the older cmake files again, except to fix something that was significantly broken.